tests/fixed/return_type_on_closures.php

```diff
@@ -1,61 +0,0 @@
-<?php
-
-function () : void {
```

Comment:
Why are we removing this? The idea of these tests was to provide examples and ensure that, regardless if we're providing the sniffs or using sniffs from a package, things are working properly (in Travis)

Reply:
Afaik these are already provided via rules from Slevomat CS (SlevomatCodingStandard.TypeHints.ReturnTypeHintSpacing) and seemed redundant to their tests, but can check again. The tests otherwise don't look like a full representation of this PR so it didn't make much sense to me to keep it.
If we want to test full CS, we can add more specific tests.

Reply:
These tests were added in cdce50da5a62b60640a43aacfa7abe8f5268ada9 to cover SpaceOnReturnTypeSniff defined by this CS. It's now replaced by external sniff.

Reply:
Actually I added the tests to validate the configuration of the ruleset and not individual sniffs (regardless of what package provides the sniffs). But sure this is up for discussion 😄 

My concern was to ensure that we're setting everything the way we want and we don't have any side effect.

Reply:
Ok, I've reverted the removal, updated it and added extra example class. 😎